### PR TITLE
add annotations + podAnnotations

### PR DESCRIPTION
**Pull request by 0xbilko**
PR for https://github.com/deckhouse/k8s-image-availability-exporter/issues/170
I also prefer to scrape via the `prometheus.io/scrape: 'true'` annotation

### CODE REVIEW

**Comment by beliys** [comment]
@nabokihms

**Review by nabokihms — CHANGES_REQUESTED** [review]
Thanks, @0xbilko. This is a great improvement. Can you bump the chart version? (it is required to automatically release a new version on artifacthub)

**Comment by 0xbilko** [comment]
@nabokihms, done

**Comment by nabokihms** [comment]
Doc is not up to date. Could you execute the make docs as well?

**Comment by 0xbilko** [comment]
@nabokihms, done

File: charts/k8s-image-availability-exporter/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: "0.9.0"
+appVersion: "0.10.0"
```
Comment (nabokihms) [inline charts/k8s-image-availability-exporter/Chart.yaml:2]:
```suggestion
appVersion: "0.9.0"
```

File: charts/k8s-image-availability-exporter/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # k8s-image-availability-exporter
 
-![Version: 0.14.0](https://img.shields.io/badge/Version-0.14.0-informational?style=flat-square) ![AppVersion: 0.9.0](https://img.shields.io/badge/AppVersion-0.9.0-informational?style=flat-square)
+![Version: 0.15.0](https://img.shields.io/badge/Version-0.15.0-informational?style=flat-square) ![AppVersion: 0.10.0](https://img.shields.io/badge/AppVersion-0.10.0-informational?style=flat-square)
```
Comment (nabokihms) [inline charts/k8s-image-availability-exporter/README.md:3]:
```suggestion
![Version: 0.15.0](https://img.shields.io/badge/Version-0.15.0-informational?style=flat-square) ![AppVersion: 0.9.0](https://img.shields.io/badge/AppVersion-0.9.0-informational?style=flat-square)
```

**Comment by nabokihms** [comment]
@0xbilko thanks!

**Pull request merged**